src/Sulu/Bundle/ContactBundle/Resources/public/js/main.js

```diff
@@ -89,21 +89,30 @@ define(['config', 'aura_extensions/iban'], function(Config, IbanExtension) {
                 ]
             });
 
+            // breadcrumbs
+            Config.set('sulucontact.breadcrumb.account', [
+                {title: 'navigation.contacts'},
+                {title: 'contact.accounts.title', link: 'contacts/accounts'}
+            ]);
+
+            Config.set('sulucontact.breadcrumb.contact', [
+                {title: 'navigation.contacts'},
+                {title: 'contact.contacts.title', link: 'contacts/contacts'}
+            ]);
+
+            // route to list
+            Config.set('sulucontact.routeToList.contact','contacts/contacts');
```

Comment:
personally i'd prefer sulucontact.route.contact.list
or is ``routeToList`` used as identifier more often?